File: medico/Domain/Medico.Domain/Models/MedicationPrescription.cs
using System;

namespace Medico.Domain.Models
{
    public class MedicationPrescription : Entity
    {
        public Guid PatientId { get; set; }
        public Patient Patient { get; set; }
        public Guid AdmissionId { get; set; }
        public Admission Admission { get; set; }
        public Guid? MedicationNameId { get; set; }
        public MedicationName MedicationName { get; set; }
        public string Medication { get; set; }
        public string Dose { get; set; }
        public string DosageForm { get; set; }
        public string Route { get; set; }
        public string Units { get; set; }
        public int Dispense { get; set; }
        public int Refills { get; set; }
        public string Sig { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: PatientPortal/src/app/patientChart/models/medicationPrescription.ts
import { GuidHelper } from 'src/app/_helpers/guid.helper';
export class MedicationPrescription {
    id: string;
    patientId: string;
    admissionId: string;
    medicationNameId: string;
    medication: string;
    dose: string;
    dosageForm: string;
    route: string;
    units: string;
    dispense: number;
    refills: number;
    sig: string;
    startDate: any;
    endDate: any;

    constructor(patientId: string = "", admissionId: string = "") {
        this.patientId = patientId;
        this.admissionId = admissionId;
        this.startDate = new Date();
    }
}